Dockerfile.webpack

```diff
@@ -0,0 +1,8 @@
+FROM node:10.15-alpine
```

Comment:
Does the builder have to be a separate docker image? Typically we try to share our images as much as possible. The Dockerfile for CB also performs `npm install`, so as far as I can tell we should be able to build the static files using a container based on that image

Reply:
@alastair Well, I just tried to take inspiration from the new setup in LB. :sweat_smile: @paramsingh @MonkeyDo what do you guys think about this?

Reply:
alastair is absolutely right! If the BD docker image can run npm commands, then there's no need to use another image.

For LB, there was previously nothing using node so the image did not contain node.